### ISSUE TITLE
bug: blink autocompletion on snippet placeholder breaks jump to next placeholder

### ISSUE THREAD

**Issue by sebiwi** [LazyVim/LazyVim#5049]
### Did you check docs and existing issues?

- [x] I have read all the LazyVim docs
- [x] I have updated the plugin to the latest version before submitting this issue
- [x] I have searched the existing issues of LazyVim
- [x] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

v0.10.2

### Operating system/version

15.1.1 (24B91)

### Describe the bug

When triggering accepting a completion on a snippet placeholder with the enter key (after inserting a snippet, and moving to a placeholder using the tab key), it is impossible to move to the next placeholder of the snippet using the tab key. A tab is inserted instead.

This works the expected way if using nvim-cmp.

![Image](https://github.com/user-attachments/assets/8221ca6f-1b86-4252-9706-0ff0c11de2dd)

### Steps To Reproduce

1. Update to the latest LazyVim version
2. Trigger a snippet insert (ctrl + space)
3. Trigger autocomplete when the cursor is on a snippet placeholder.
4. Select an autocomplete proposition with <ctrl-n> then <cr>.
5. Press Tab to switch to the next snippet placeholder.

A tab is inserted then.

### Expected Behavior

1. Update to the latest LazyVim version
2. Trigger a snippet insert (ctrl + space)
3. Trigger autocomplete when the cursor is on a snippet placeholder.
4. Select an autocomplete proposition with <ctrl-n> then <cr>.
5. Press Tab to switch to the next snippet placeholder.
6. The cursor moves to the next snippet placeholder.

### Repro

```lua
I couldn't reproduce this with repro.lua, but I could with the starter config (https://github.com/LazyVim/starter).
```

**Comment by folke** [comment]
https://github.com/Saghen/blink.cmp/issues/289

**Comment by sebiwi** [comment]
Just out of curiosity, is everyone using blink encountering this issue then? Is there a workaround, other than not using snippets?

**Comment by chaozwn** [comment]
> Just out of curiosity, is everyone using blink encountering this issue then? Is there a workaround, other than not using snippets?

nvim 0.10.x is ok.

**Comment by sebiwi** [comment]
@folke after reading the [blink issue you shared](https://github.com/LazyVim/LazyVim/issues/5049#issuecomment-2542524545) thoroughly, unless I misunderstood something, I think that is not the issue I'm encountering. 

That issue, as @chaozwn said, is about:

> In snippet mode, that is, when you select a snippet item during completion, if the snippet_forward is not fully cycled through, my Tab key won't work(It just performs a normal jump without selecting any item from the completion menu). This happens even when I use the default configuration of LazyVim. Removing the fallback results in the same effect.

My issue is about:

> After inserting a snippet, and moving to a placeholder (tabstop) using the tab key, when triggering autocompletion (ctrl + leader) and accepting a completion proposition with the enter key (<cr>), it is impossible to move to the next placeholder of the snippet using the tab key. A tab is inserted instead.

Also, as @chaozwn said, that issue should not be present on nvim 0.10.x, and I'm using 0.10.2.

I updated the issue description, it may have not been clear enough before.

Could it be possible to re-open this issue please?

Thanks!

**Comment by sebiwi** [comment]
This was fixed on v14.5.0, no need to reopen the issue.

Thanks!
